data/json/items/gun/762R.json

```diff
@@ -0,0 +1,112 @@
+[
+  {
+    "id": "mosin44",
+    "type": "GUN",
+    "name": "Mosin-Nagant M44",
+    "name_plural": "Mosin-Nagant M44",
+    "description": "The Model 44 is the carbine sized variant of the standard 1891/30.  It has a shorter barrel, therefore it is weaker than the 1891/30, but easier to carry.",
+    "weight": 4100,
+    "volume": 10,
+    "price": 23000,
+    "bashing": 12,
+    "to_hit": -1,
+    "material": [ "steel", "wood" ],
+    "symbol": "(",
+    "color": "brown",
+    "skill": "rifle",
+    "ammo": "762R",
+    "dispersion": 30,
+    "sight_dispersion": 60,
+    "aim_speed": 6,
+    "durability": 10,
+    "loudness": 150,
+    "clip_size": 5,
```

Comment:
If you can define how they are supposed to work I'll have a go at implementing them. I presume speedloaders for revolvers would work similarly?